test/onigumo_cli_test.exs

```diff
@@ -96,9 +116,9 @@ defmodule OnigumoCLITest do
       String.match?(output, ~r/\N\n\z/)
     end
 
-    defp usage_message_printed?(function) do
+    defp usage_message_printed?(function, reason) do
       output = capture_io(function)
-      String.starts_with?(output, "onigumo: invalid usage")
+      String.starts_with?(output, "onigumo: invalid usage – #{reason}")
```

Comment:
Just thinking out loud.

```suggestion
      message = "onigumo: invalid usage – #{reason}"
      String.starts_with?(output, message)
```

Reply:
yes it is more clean suppose

Reply:
but then it is for another PR since the same way of solving is for help message